automata/fa/dfa.py

```diff
@@ -702,25 +830,49 @@ def _find_state(
         return any(target_state_fn(state) for state in bfs_states)
 
     @staticmethod
-    def _cross_product(lhs: DFA, rhs: DFA) -> Tuple[DFAStateT, ExpandStateFn]:
-        """Builds the cross product between the two DFAs"""
+    def _cross_product(
+        lhs: DFA, rhs: DFA, lhs_relevant: BooleanLiteral, rhs_relevant: BooleanLiteral
```

Comment:
@EduardoGoulart1 @eliotwrobson For clarity, I request that the two new parameters at the end be keyword-only. And, as I mentioned in my above comment, you should be able to just use `bool` for the type hints. That is:

```python
def _cross_product(lhs: DFA, rhs: DFA, *, lhs_relevant: bool, rhs_relevant: bool) -> Tuple[DFAStateT, ExpandStateFn]:
    # ...
```

Even though the method is internal, this will make it much clearer later in the code where you are currently passing `True, True` to this method.

Reply:
@caleb531 agreed, done here! https://github.com/caleb531/automata/pull/147/commits/070ff3206873feee4728c796792b87ff9d88914f